docs/integrations/sources/harvest-migrations.md

```diff
@@ -0,0 +1,30 @@
+# Harvest Migration Guide
+
+## Upgrading to 1.0.0
+
+This update results in a change the following streams, requiring reset:
+- `expenses_clients`
+- `expenses_categories`
+- `expenses_projects`
+- `expenses_team`
+- `time_clients`
+- `time_projects`
+- `time_tasks`
+- `time_team`
+- `uninvoiced`
+- `estimate_messages`
+- `invoice_payments`
+- `project_assignments`
+
+We're continuously striving to enhance the quality and reliability of our connectors at Airbyte.  As part of our commitment to delivering exceptional service, we are transitioning Source Harvest from the Python Connector Development Kit (CDK) to our a new low-code framework improving maintainability and reliability of the connector. However, due to differences between the Python and low-code CDKs, this migration constitutes a breaking change.
```

Comment:
there is a typo::

We're continuously striving to enhance the quality and reliability of our connectors at Airbyte.  As part of our commitment to delivering exceptional service, we are transitioning Source Harvest from the Python Connector Development Kit (CDK) to our new low-code framework improving maintainability and reliability of the connector. However, due to differences between the Python and low-code CDKs, this migration constitutes a breaking change.

Reply:
Updated.